### DISCUSSION TITLE
Most recent VDH update fails every download attempt

### DISCUSSION THREAD

**Discussion by OjciecTom** [aclap-dev/video-downloadhelper#316] [Help and Q&A]
### Your question

I have tried to download numerous videos from multiple sites. 
Every single one fails immediately upon selecting the video from the dropdown list and clicking "Download" or "Download and convert" or "Quick Download".

The error started appearing right after the last automatic Video Download Helper update about two weeks ago or so.

I never had a problem before this.

I have tried uninistalling and reinstalling VDH several times, and have kept up with Firefox updates.

The ERROR MESSAGE (same basic message for every failed download)

moz-extension://e9c79430-6cf6-4d1f-ab7a-c8c0f0b08259/background/main.js:1
receive@moz-extension://e9c79430-6cf6-4d1f-ab7a-c8c0f0b08259/background/main.js:1:13157
Zp</callCatchAppNotFound/</<@moz-extension://e9c79430-6cf6-4d1f-ab7a-c8c0f0b08259/background/main.js:1:42626

SAMPLE URL that fails:  https://www.youtube.com/watch?v=BGXTt9hQbic
But they all fail.

Operating system:  Windows 10 Home 64bit Version	10.0.19045 Build 19045

### Addon & CoApp version and details

VDH data:

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 89b5f870090e412d
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\fathe",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
If they all fail it's probably a local config problem. Maybe an antivirus or firewall blocking issue.
